### PR TITLE
In readme, fix Table of contents, add platform support, getting started, and features list.

### PR DESCRIPTION
**Pull request by RobbieMcKinstry**

### CODE REVIEW

**Comment by RobbieMcKinstry** [comment]
@EricGhildyal If you'd like to make provide any feedback, happy to accept it! Just wanted to get this merged sooner rather than later.

File: README.md [hunk 1]
```diff
@@ -10,9 +10,52 @@
...
+1. First, make sure your AWS credentials are configured:
+   ```bash
+   $ aws sts get-caller-identity
+   ```
+
+   If you're credentials aren't set, either login with `aws sso login --profile my-profile` or set your Access Key Id and Secret Access Key: 
+   ```bash
+   $ export AWS_ACCESS_KEY_ID="${MY_ACCESS_KEY}"
+   $ export AWS_SECRET_ACCESS_KEY="${MY_SECRET_ACCESS_KEY}"
+   $ export AWS_REGION="us-west-2"
```
Comment (EricGhildyal) [inline README.md:26]: Nit: `us-east-1` or `us-east-2` are common in tutorials since they cost less

**Review by EricGhildyal — COMMENTED** [review]
LGTM!

File: README.md (hunk 1 above)
Comment (RobbieMcKinstry) [reply README.md:26]: Oh, they do!?! I thought all regions cost the same! I thought we might give ourselves away as East Coasters if we used us-east-2 lmao That makes more sense!

File: README.md [hunk 2]
```diff
@@ -10,9 +10,52 @@
...
+## Getting Started
+
+This section explains how to deploy an AWS Lambda function to AWS API Gateway using MultiTool Canary. Just point MultiTool Canary to a pre-built Lambda zip, tell it which API Gateway, Stage, and Lambda to deploy to, and it will take care of the rest.
+
+1. First, make sure your AWS credentials are configured:
+   ```bash
+   $ aws sts get-caller-identity
+   ```
+
+   If you're credentials aren't set, either login with `aws sso login --profile my-profile` or set your Access Key Id and Secret Access Key:
```
[the remark this answers is not in the document]
Comment (RobbieMcKinstry) [reply README.md:22]: Ugh

**Pull request merged**